### DISCUSSION TITLE
2022-01-31 PyAerocom meeting

### DISCUSSION THREAD

**Discussion by avaldebe** [metno/pyaerocom#577] [Development meetings]
## Follow up from last meeting

Jan 24th #572

## New Issues

- #551 
- #573: bug
- #574: bug, PR #575

## New PRs

- #552: needs review
- #575: ready to merge
- #576

## Old(er) PRs

- #546
- #547: needs more work, could be merged if scope is reduced
- #576
- #550

## Move to GH Discussions?

...

## For next meeting

- [ ] Daniel and Álvaro will resolve and merge PR #546
- [ ] Python packaging mini workshop with Álvaro, Krister, Ove, Daniel and Charlie
- [ ] Jan will present what he learned from #550

**Comment by jgriesfeller** [comment]
We should also discuss why CI is failing so often and how to tackle that

**Reply by avaldebe** [reply]
yes, we should.
So far we have CI failing because:
- new warnings: PR #552
- fail to download test data set
- fail format check job: configure VS code to [format and lint on save]
- other?

[format and lint on save]: https://realpython.com/advanced-visual-studio-code-python/#setting-format-and-lint-on-save

**Reply by jgriesfeller** [reply]
I have just looked at the apache logs at the webserver. There is no indication at all that the test data is not available. It's also on the local filesystem, so no access to Lustre. I somehow doubt that the webserver is the problem. There's less than 50 accesses to testdata per day (but a lot more the the htap masks).
I will check if there's the IP of the runner in the CI log. Then I can look if it's getting through.

**Reply by avaldebe** [reply]
https://github.com/metno/pyaerocom/runs/4937910156?check_suite_focus=true

fails to write an aeroval test dataset

```
PermissionError: [Errno 13] Permission denied: b'/home/runner/tmp/pyatest/aeroval/coldata/test/exp1/TM5-AP3-CTRL/od550aer_od550aer_MOD-TM5-AP3-CTRL_REF-AERONET-Sun_20100101_20101231_daily_WORLD-wMOUNTAINS.nc'
```
